app/profiles/[landlordId]/page.tsx

```diff
@@ -0,0 +1,103 @@
+import { createClient } from "@/utils/supabase/server"
+import { cookies } from "next/headers"
+import { notFound } from "next/navigation"
+
+export default async function landlordProfilePage( {params }: { params: { landlordId: string } }) {
+    // check if a landlord id was provided
+    const landlordId = params.landlordId
+    if (!landlordId) return (
+        <div>
+            <h1>ERROR: No Landlord Id provided</h1>
+        </div>
+    )
```

Comment:
I believe this check is un-necessary, as the case where there is no landlord id provided will go to the other page you created (or 404 if that one wan't defined)